.github/renovate.json

```diff
@@ -30,7 +31,7 @@
     },
     {
       "groupName": "golang-lang",
-      "matchPackagePatterns": ["^golang$", "xgo"],
```

Comment:
Why have you removed "xgo" from the group here? IMO xgo and golang updates should still be grouped.

Reply:
Ah, not on purpose. Thanks for catching!